app/controllers/api/v1/search_controller.rb

```diff
@@ -525,6 +536,21 @@ def sanitize_search_values(terms)
         end
       end
 
+      # extract any "quoted phrases" from query string and tokenize terms
+      def self.extract_phrases_from_search(query_string:)
+        terms = []
+        query_string.split("\"").each do |substring|
+          # when splitting on double quotes, phrases will not have any leading/trailing whitespace
```

Comment:
do we need to escape regex special characters that are inside terms?

Reply:
That's already happened in https://github.com/broadinstitute/single_cell_portal_core/blob/jb-keyword-phrase-search/app/controllers/api/v1/search_controller.rb#L147 - `sanitize_search_terms` escapes any unsafe characters.

Reply:
I actually just realized that this isn't true in this case - it's removing some unsafe characters, but not ones specific to regular expressions.  We want to use `Regexp.quote` for that.  I'll make that change.

Reply:
c9a63d2